Extension/test/scenarios/SingleRootProject/tests/lmTool.test.ts

```diff
@@ -0,0 +1,366 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
```

Comment:
When are these tests run? Or how do I run them?

Reply:
These are non unit test as they have dependency on vscode (for shared types). We don't seem to have any pipelines running these scenarios tests for some reason. I run it from command line.

`yarn test --no-sandboax --disable-updates --skip-welcome --skip-release-notes --disable-extensions --extensionDevelopmentPath=.\ --extensionTestsPath=dist\test\common\selectTests --scenario=SingleRootProject SingleRootProject`

![image](https://github.com/user-attachments/assets/b3aa1e93-34dc-4677-8e9f-5619cfa27e34)
![image](https://github.com/user-attachments/assets/aad69cfd-4117-4c31-a91b-46dcfb9e8394)
![image](https://github.com/user-attachments/assets/ef929f5f-700b-48b2-9617-9ea52eecb304)

I can also debug these as well.

![image](https://github.com/user-attachments/assets/abbb9d86-609e-4c9c-9d24-271f6343850e)

![image](https://github.com/user-attachments/assets/2ed8c5c3-ae30-425c-909b-1343b0f9d99a)

Reply:
Ah, okay, I thought I had tried running the VSCode Tests with SingleRootProject, but I must have done something wrong prevoiusly.